### DISCUSSION TITLE
New release on App Store

### DISCUSSION THREAD

**Discussion by dvidebaek** [InfiniTimeOrg/InfiniLink#125] [General]
Hi all,
When will a new version of the app be released on the App Store? The latest update is over a year old

**Comment by liamcharger** [comment]
Hi @dvidebaek, I plan on releasing a new version as soon as all the newly implemented features are ready to go. I believe the last thing that needs to be implemented is fetching the forecast for the new Weather app in InfiniTime, and the last two live watchfaces. 

Thanks for your patience!

**Reply by NitroNils** [reply]
Congratulations on the beta release! Is there a TestFlight version somewhere to access the beta without building it ourselves?

**Reply by liamcharger** [reply]
It will be available on TestFlight as soon as the community manager gets the details needed for publish! I’ll comment in this Discussion when it is.
